src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -2770,12 +2770,12 @@ namespace ns1
             // TODO (tomat): EOFUnexpected shouldn't be reported if we enable parsing global statements in namespaces
             DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(test,
                 // (4,5): error CS1022: Type or namespace definition, or end-of-file expected
-                // (4,10): error CS0116: A namespace does not directly contain members such as fields or methods
+                // (4,10): error CS0116: A namespace or a top-level entry point cannot directly contain members such as fields or properties
```

Comment:
>// (4,10): error CS0116: A namespace or a top-level entry point cannot directly contain members such as fields or properties [](http://example.com/codeflow?start=16&length=124)

It looks like the original wording of the message was incomplete in the base-line: "A namespace cannot directly contain members such as fields, methods **or statements**". I assume that this error is reported for ```goto Labl;```, which is a statement (BTW, I think it would be good to include comments with the corresponding source as it is done in the test below and make it clear what comment belongs to what ```ErrorDescription```). It feels like we should find a wording that accommodates this scenario (a misplaced statement) as well